tests/models/propainter/test_modeling_propainter.py

```diff
@@ -0,0 +1,238 @@
+        self.image_size = image_size
+        self.num_channels = num_channels
+        self.is_training = is_training
+        self.hidden_size = hidden_size
+        self.num_hidden_layers = num_hidden_layers
+        self.num_attention_heads = num_attention_heads
+        self.num_frames = num_frames
+        # self.hidden_act = hidden_act
+        # self.hidden_dropout_prob = hidden_dropout_prob
+        # self.attention_probs_dropout_prob = attention_probs_dropout_prob
+        # self.type_sequence_label_size = type_sequence_label_size
+        # self.initializer_range = initializer_range
+
+    def prepare_config_and_inputs(self):
+        pixel_values = floats_tensor([self.batch_size, self.num_frames, 3, self.image_size, self.image_size])
+
+        pixel_values_inp = pixel_values.cpu.numpy()
+        pixel_values_inp = [[pixel_values_inp[b, t].transpose(1, 2, 0) for t in range(pixel_values_inp.shape[1])] for b in range(pixel_values_inp.shape[0])]
+
+        masks = ids_tensor([self.batch_size, self.num_frames, 1, self.image_size, self.image_size], vocab_size=2).float()
+        flow_masks = masks_dilated = masks
+        config = self.get_config()
+
+        return config, pixel_values_inp, pixel_values, flow_masks, masks_dilated, (self.image_size, self.image_size)
+
+    def get_config(self):
+        return ProPainterConfig(
+            num_channels=self.num_channels,
+            hidden_size=self.hidden_size,
+            num_hidden_layers=self.num_hidden_layers,
+            num_attention_heads=self.num_attention_heads,
+        )
+
+    def create_and_check_model(self, config, pixel_values_inp, pixel_values, flow_masks, masks_dilated, size):
+        model = ProPainterModel(config=config)
+        model.to(torch_device)
+        model.eval()
+        result = model(pixel_values_inp, pixel_values, flow_masks, masks_dilated, size)
+        self.parent.assertEqual(result.reconstruction.shape, (self.batch_size, self.num_frames, self.image_size, self.image_size, 3))
+
+    def prepare_config_and_inputs_for_common(self):
+        config_and_inputs = self.prepare_config_and_inputs()
+        (
+            config,
+            pixel_values_inp,
+            pixel_values,
+            flow_masks,
+            masks_dilated,
+            size,
+        ) = config_and_inputs
+        inputs_dict = {"pixel_values_inp": pixel_values_inp, "pixel_values": pixel_values, "flow_masks": flow_masks, "masks_dilated": masks_dilated, "size": size}
+        return config, inputs_dict
+
+
+@require_torch
+class ProPainterModelTest(ModelTesterMixin, PipelineTesterMixin, unittest.TestCase):
+    """
+    Here we also overwrite some of the tests of test_modeling_common.py, as ProPainter does not use input_ids, inputs_embeds,
+    attention_mask and seq_length.
+    """
+
+    all_model_classes = (
+        (
+            ProPainterModel,
+        )
+        if is_torch_available()
+        else ()
+    )
+    fx_compatible = False
```

Comment:
yes the modeling test is giving this error when its `True` : `FAILED tests/models/propainter/test_modeling_propainter.py::ProPainterModelTest::test_torch_fx - NotImplementedError: Model ProPainterModel is not supported yet, supported models: AlbertForMas...
`
But i still removed the statement because without it also the test is passing.